src/app/global-header/global-header.component.html

```diff
@@ -1,14 +1,48 @@
-<div class="head-title text-center pb-2 mt-2" *ngIf="showUsername">
-  {{ "HI" | translate }}
-  <span class="text-blue-shade0">&nbsp;{{authService?.currentUser?.name | titlecase}}</span>
-</div>
 <div class="text-center">
-  <div class="row px-4">
+  <div class="row">
     <div class="col-12" [ngClass]="{ 'card mb-3 linear-bg': showBackground }">
-      <img class="img-fluid" src="../../assets/images/headerlogo1.png" alt="header-logo">
+      <div class="row g-0 px-3">
+        <div class="col-3 col-md-1 px-0 pr-2">
+          <svg
+            width="74"
+            height="69"
+            viewBox="0 0 74 69"
```

Comment:
@shreyas1434shinde dont use svg directly into the html, save svg file in images folder and use `img` tag to render it

Reply:
@itsvick comment resolved